### PR TITLE
Logger package

### PR DESCRIPTION
**Pull request by DrAnax**
Moved global logger into a separate package

### CODE REVIEW

File: server/src/game_player.go [hunk 1]
```diff
@@ -189,15 +191,14 @@ func (p *GamePlayer) PlayCard(c *Card) {
 	// Update the progress
 	progressFloat := float64(g.Score) / float64(g.MaxScore) * 100 // In percent
 	progress := int(math.Round(progressFloat))
-	oldProgress := t.Progress
```
Comment (Zamiell) [inline server/src/game_player.go:192]: what does this have to do with logging?

Comment (DrAnax) [reply server/src/game_player.go:192]: That's a fix from linter warnings (ifshort)

File: server/src/http_google_analytics.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func httpGoogleAnalytics(c *gin.Context) {
 	if cookie, err := r.Cookie("_ga"); err != nil {
 		// They don't have a cookie set, so set a new one
 		clientID = uuid.NewV4().String()
-		http.SetCookie(w, &http.Cookie{ // nolint: exhaustivestruct
+		http.SetCookie(c.Writer, &http.Cookie{ // nolint: exhaustivestruct
```
Comment (Zamiell) [inline server/src/http_google_analytics.go:43]: looks like there are many changes in this PR that have nothing to do with refactoring a logger package

Comment (DrAnax) [reply server/src/http_google_analytics.go:43]: Same as above

Comment (Zamiell) [reply server/src/http_google_analytics.go:43]: instead of changing code, just set the linter version to one from 2019
